src/status_im/contexts/wallet/collectible/view.cljs

```diff
@@ -28,13 +29,17 @@
      collection-name]]])
 
 (defn cta-buttons
-  []
+  [collectible]
   [rn/view {:style style/buttons-container}
    [quo/button
     {:container-style style/send-button
      :type            :outline
      :size            40
-     :icon-left       :i/send}
+     :icon-left       :i/send
+     :on-press        #(rf/dispatch
```

Comment:
🙌 
(also this is one reason it's good to test. It's technically new functionality 👍